addons/mail/static/src/core/common/attachment_view.js

```diff
@@ -73,13 +71,25 @@ export class AttachmentView extends Component {
     }
 
     popoutAttachment() {
-        this.mailPopoutService.popout(this.__owl__.bdom.parentEl).focus();
+        class PopupWrapper extends Component {
+            static template = xml`<div class="o_attachment_preview" style="display: block;"><AttachmentView t-props="props"/> </div>`;
+            static components = {
+                AttachmentView,
+            };
+        }
+
+        const app = new App(PopupWrapper, {
+            name: "Popup",
+            env: this.env,
+            props: this.props,
+            getTemplate,
+        });
+        this.mailPopoutService.popout(app, this.toggleVisibility.bind(this));
```

Comment:
This should be in the service not here, like in dialog service
Also, it should be a props validation.